deploy/rbd/kubernetes/csi-rbdplugin-provisioner.yaml

```diff
@@ -126,6 +126,7 @@ spec:
             - "--rbdhardmaxclonedepth=8"
             - "--rbdsoftmaxclonedepth=4"
             - "--enableprofiling=false"
+            - "--clustername=k8s-cluster-1"
```

Comment:
let's not add this to default deployment as it does not make sense to set dummy clusterName

Reply:
@Madhu-1 I thought we discussed this earlier, this will be helpful at e2e ?

Reply:
Need to set in the code e2e code. As you are adding this to the default templates and helm charts, lt will get applied to all clusters which are deployed by users.

Reply:
@Madhu-1 
In [`createORDeleteRbdResources()` ](https://github.com/ceph/ceph-csi/blob/devel/e2e/rbd.go#L132) we directly feed yaml filenames to `r.Do(action)`. To avoid rework let me know how we would like to edit the provisioner.yaml both for helm and non-helm deployments?

@nixpanic thoughts?

Reply:
You can add a new option to the `yamlResourceNamespaced` type, and use that to set the cluster name for the provisioner? This can be used for the non-Helm deployment, you can make the test that validates it depend on it.

Or, what also works for the Helm case, get the Deployment through the k8s API, and update the args by adding the `--clustername` option and wait for the pod(s) to be restarted.